### PR TITLE
Sorted News Articles

### PR DESCRIPTION
**Pull request by anks321**
Sorted the news articles from latest to oldest. Issue #833

### CODE REVIEW

File: pages/news.js [hunk 1]
```diff
@@ -1,57 +1,93 @@
...
-  for (let pageNumber = 1; pageNumber <= pageCount; pageNumber += 1) {
-    if (pageNumber === page) {
-      pageLinks.push(
-        <PageNumber className="current" key={pageNumber}>
-          {pageNumber}
-        </PageNumber>
-      );
-    } else {
-      const pagePath = `/news?page=${pageNumber}`;
+  // const pageLinks = [];
```
Comment (LayaTaal) [inline pages/news.js:32]: Can you remove the commented code before merging?

File: components/homepage/HomepageNewsFeed.js [hunk 2]
```diff
@@ -5,14 +5,27 @@ import Image from 'next/image';
...
+import content from '../../content/newsfeed.md';
 
 // Load images
 import facebook from '../../images/tji-fb-logo-blue.svg';
 import twitter from '../../images/tji-twitter-logo-blue.svg';
 import github from '../../images/tji-github-logo-blue.svg';
 
 class HomepageNewsFeed extends React.Component {
-  render() {
+  render() {git
```
Comment (LayaTaal) [inline components/homepage/HomepageNewsFeed.js:16]: Looks like a stray `git` got committed here. Once you remove this, we should be good to go here!

**Review by LayaTaal — CHANGES_REQUESTED** [review]
Just a couple of small changes and this will be ready to merge! Thank you for jumping in to help.

**Comment by anks321** [comment]
I have made the requested changes

**Comment by anks321** [comment]
Can you check if the error has been resolved or not and merge the pr?

**Comment by LayaTaal** [comment]
> Hi can you merge this pr . Or inform me if theres something wrong with the current code

Apologies, I was out of town the past week and couldn't get to this. I will review it today and let you know if there are any other changes needed.

File: pages/news.js [hunk 3]
```diff
@@ -1,76 +1,77 @@
...
 import { NextSeo } from 'next-seo';
 import { useRouter } from 'next/router';
-import Link from 'next/link';
+
 import Layout from '../components/Layout';
 import Primary from '../components/Primary';
 import Sidebar from '../components/Sidebar';
 import NewsFeed from '../components/NewsFeed';
 import content from '../content/newsfeed.md';
+import Pagelinks from '../components/Pagelinks';
```
Comment (LayaTaal) [inline pages/news.js:11]: Check this out real quick. If you update the filename to be capitalized, this should work then.

**Review by LayaTaal — CHANGES_REQUESTED** [review]
Thanks for continuing to work through this! Please check the eslint error and I'll be happy to give it another look to try and get this merged.

File: pages/news.js (hunk 3 above)
Comment (anks321) [reply pages/news.js:11]: Done , should be fine now

**Review by LayaTaal — APPROVED** [review]
Looks perfect, thank you!

**Pull request merged**